lib/generateMarkdown.js

```diff
@@ -386,15 +386,14 @@ function getEnumString(schema, type, depth) {
     var propertyEnumNames = schema['gltf_enumNames'];
 
     var allowedValues = '';
-    var prefix = '\n' + ('      '.substring(0, depth * 3)) + '* ';
     var length = propertyEnum.length;
     for (var i = 0; i < length; ++i) {
         var element = style.enumElement(propertyEnum[i], type);
         if (defined(propertyEnumNames)) {
             element += " " + propertyEnumNames[i];
         }
 
-        allowedValues += prefix + element;
+        allowedValues += style.bulletItem(element.toString(), depth);
```

Comment:
I took a look at the code for `style.bulletItem` and it looks like it has a bug.  It does all that indentation logic, and then it just drops it at the end (it doesn't do a `+=` when adding the bullet and item content, but rather just an `=`).

So...you should probably fix `style.bulletItem` with this PR, and you may as well simplify the logic to use `.repeat` as opposed to the `for loop`.